src/api/spec/bootstrap/features/webui/users_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'browser_helper'
+
+RSpec.feature 'Bootstrap_User Contributions', type: :feature, js: true do
+  let!(:user) { create(:confirmed_user) }
+
+  context 'no contributions' do
```

Comment:
We should have a spec with at least some contributions.

Reply:
done